src/qforte/sq_operator.h

```diff
@@ -59,10 +55,16 @@ class SQOperator {
     /// The linear combination of second quantized operators. Stored in pairs of
     /// coefficients, and then a vector of N created indices, followed by N annihilated indices.
     /// Orbital indices start at zero.
-    std::vector<std::pair< std::complex<double>, std::vector<size_t>>> terms_;
+    std::vector<std::tuple< std::complex<double>, std::vector<size_t>, std::vector<size_t>>> terms_;
 
     /// Calculate the parity of permutation p
-    bool permutive_sign_change(std::vector<int> p);
+    bool permutive_sign_change(std::vector<int> p) const;
```

Comment:
This does not really belong here since it is a general function. I would suggest to move to a `combinatorial.cc` or `math.cc` file. This can be done in another PR.
```suggestion
    bool permutation_sign(std::vector<int> p) const;
```

Reply:
Yeah, let's defer moving this to another PR, when we have a better idea what this other function might look like. Done the renaming, though.